docs/using.md

```diff
@@ -234,41 +246,57 @@ def callback_handler (stream_consumer: qx.StreamConsumer, data: qx.TimeseriesDat
+!!! tip
 
-The following example code demonstrates a simple use of `LocalFile Storage`:
+    For this facility to work in Quix Platform you need to enable the State Management feature. You can enable it in the `Deployment` dialog, where you can also specify the size of storage required. When using Quix Streams with a third-party broker such as Kafka, no configuration is required, and data is automatically stored on the local file system.
+
+## Tracking running totals across multiple streams
+
+Sometimes you want to track a running total across all streams in a topic. The problem is that when you scale using replicas, there is no way to share data between all replicas in a consumer group. 
+
+The solution is to write the running total per stream (with stream ID) to an output topic. You can then have another processor in the pipeline to calculate total values from inbound messages. The following code demonstrates how to do this:
 
 ```python
-my_var = qx.InMemoryStorage(qx.LocalFileStorage())
-...
-topic_consumer.on_stream_received = on_stream_received_handler
-topic_consumer.on_committed = my_var.flush
 ...
-```
+g_running_total_per_stream = qx.InMemoryStorage(qx.LocalFileStorage())
 
-This ensures that the variable `my_var` is persisted, as periodically (default is 20 seconds) it is flushed to local file storage.
+def callback_handler (stream_consumer: qx.StreamConsumer, data: qx.TimeseriesData):
 
-If the system crashes (or is restarted), Kafka resumes message processing from the last committed message. This facility is built into Kafka.
+    if stream_consumer.stream_id not in g_running_total_per_stream:
+        g_running_total_per_stream[stream_consumer.stream_id] = 0
+    
+    ...
 
-!!! tip
+    g_running_total_per_stream[stream_consumer.stream_id] += some_value
+    data.add_value("RunningTotal", g_running_total_per_stream[stream_consumer.stream_id])
+
+    topic_producer.get_or_create_stream(stream_consumer.stream_id).timeseries.publish(data)
+...
+```
+
+In this case the running total is published to its own stream in the output topic. The next service in the data processing pipeline would be able to sum all running totals across all streams in the output topic.
 
-    For this facility to work in Quix Platform you need to enable the State Management feature. You can enable it in the `Deployment` dialog, where you can also specify the size of storage required. When using Quix Streams with a third-party broker such as Kafka, no configuration is required, and data is stored on the local file system.
+Also, in this example, the running total is persisted in file storage, and so is preserved in the event of service restarts and system crashes.
 
 ## Conclusion
 
 In this topic you have learned:
```

Comment:
"topic" word again

Reply:
Yes, I will change it! :)

Reply:
Changed. I've also fixed a couple of other occurrences in another PR on the main docs repo. I've also updated our writing style guide and best practice guide accordingly. Good shout, thanks!